test/runtests.jl

```diff
@@ -2,8 +2,13 @@ using DiffEqBiological
 using Base.Test
 
 tic()
-@time @testset "Gillespie Tests" begin include("gillespie.jl") end
-@time @testset "Variable Rate Reaction Tests" begin include("variable_rate_reactions.jl") end
-@time @testset "@reaction_network Tests" begin include("reaction_network.jl") end
-@time @testset "Higher order reaction Tests" begin include("higher_order_reactions.jl") end
+#@time @testset "Gillespie Tests" begin include("gillespie.jl") end
+#@time @testset "Variable Rate Reaction Tests" begin include("variable_rate_reactions.jl") end
```

Comment:
You should make these tests, when applicable, use the new DSL to make sure that the old models can still be solved.